python/GafferSceneUI/SelectionToolUI.py

```diff
@@ -52,6 +52,7 @@
 		- Drag to PathFilter or Sets node to add/remove their paths
 	""",
 
+	"viewer:shortCut", "Q",
```

Comment:
Is "Q" standard? I was guessing "S" for some reason. Maybe "Q" matches Maya or somthing? Can we add this to the tooltip?